airflow/models.py

```diff
@@ -71,7 +71,11 @@
 from airflow.utils.timeout import timeout
 from airflow.utils.trigger_rule import TriggerRule
 
-Base = declarative_base()
+SQL_ALCHEMY_SCHEMA = configuration.get('core', 'SQL_ALCHEMY_SCHEMA')
+if SQL_ALCHEMY_SCHEMA is None or SQL_ALCHEMY_SCHEMA.isspace():
```

Comment:
Without the defaults dictionary, the result of configuration.get should always be a (possibly empty) string and I think you should be able to change this test to simply `if SQLALCHEMY_SCHEMA:`, but please test to double check.